poc/test_vectors.sage

```diff
@@ -74,26 +74,48 @@ def file_json(h2c, vectors, path="vectors"):
 
 
 def file_ascii(h2c, vectors, path="ascii"):
+    data = ""
     with open(path + "/" + h2c.suite_name + ".txt", 'wt') as f:
-        f.write(Printer.tv.text("suite", h2c.suite_name) + "\n")
-        f.write(Printer.tv.text("dst", h2c.dst) + "\n")
+        data = data + Printer.tv.text("suite", h2c.suite_name) + "\n"
+        data = data + Printer.tv.text("dst", h2c.dst) + "\n"
         for vec in vectors:
-            f.write("\n" + Printer.tv.text("msg", vec["msg"]) + "\n")
-            f.write(Printer.tv.point("P", vec["P"]) + "\n")
+            data = data + "\n" + Printer.tv.text("msg", vec["msg"]) + "\n"
+            data = data + Printer.tv.point("P", vec["P"]) + "\n"
             for idx in range(0, len(vec["u"])):
-                f.write(Printer.tv.gf("u[%d]" % idx, vec["u"][idx]) + "\n")
+                data = data + Printer.tv.gf("u[%d]" % idx, vec["u"][idx]) + "\n"
             if h2c.is_ro:
-                f.write(Printer.tv.point("Q0", vec["Q0"]) + "\n")
-                f.write(Printer.tv.point("Q1", vec["Q1"]) + "\n")
+                data = data + Printer.tv.point("Q0", vec["Q0"]) + "\n"
+                data = data + Printer.tv.point("Q1", vec["Q1"]) + "\n"
             else:
-                f.write(Printer.tv.point("Q", vec["Q"]) + "\n")
+                data = data + Printer.tv.point("Q", vec["Q"]) + "\n"
+        f.write(data)
+    return data
 
 
-def create_files(suite):
+def combined_ascii(results, path="ascii"):
```

Comment:
Does this combined_ascii file get used somewhere? Is the idea that this replaces the `poc/test_vectors_appx.py` script?

Reply:
Doh. I didn't even realize that was a thing!

Reply:
No strong preference. The order of the sections is slightly different, which ends up making the diff to the document huge. So I guess that's a small thing in favor of keeping the old script, but only a very small one.

Reply:
I'll remove it :)